infrastructure/docker/docker-compose.docker-for-x.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.7'
+
+services:
+    router:
+        network_mode: "${PROJECT_NAME}_default"
```

Comment:
wat?

Reply:
Does not looks like a valid value: https://docs.docker.com/compose/compose-file/#network_mode

Reply:
I don't know what to tell you. This is the default value when you don't specify it.
Try to remove the value, and run `docker inspect app_router_1`. This is how I get the value.

Anyway, it works well on my linux. It should be the same on other system. But please, test it on windows and tell me if I'm wrong. Thanks :pray:

Reply:
For the record, the full explanation: 

When using docker composer, docker composer create a network and put all containers in it. The format of the network name is the following `[APP_NAME]_default`. `default` because this is the default one...

So here I'm reproducing what docker-compose does, but manually.

And this value is totally valid :) I guess you missed the following sentence:

>Network mode. Use the same values as the docker client --network parameter, plus the special form service:[service name].